src/pages/insight.py

```diff
@@ -186,47 +193,3 @@ def show_insight_details(pathname, custom_insights):
   Input('download-insight-button', 'n_clicks'),
   prevent_initial_call=True,
 )
-
-
-@callback(
-  Output('insight-pdf-download', 'data'),
-  Output('notification-container', 'sendNotifications', allow_duplicate=True),
-  Output('download-insight-button', 'loading'),
-  Input('download-insight-button', 'n_clicks'),
-  State('custom-insights-store', 'data'),
-  State('url', 'pathname'),
-  prevent_initial_call=True,
-)
-def handle_click_download(n_clicks, custom_insights, pathname):
```

Comment:
Why was this removed, is it being done somewhere else now? I don't see any other callbacks where we're downloading the PDF